src/Migrations/Version20250103110506.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BigGive\Identity\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Create a table for global persisted variables and add one for use in one-off person copy job
+ */
+final class Version20250103110506 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'CREATE KeyValue table for persisted globals';
```

Comment:
I would probably have used Redis as it's already available, but can always mix and match or drop this table after the one time job is done, if we think we would go for Redis in future similar cases.

Reply:
Right, I thought about Redis, I think I mostly preferred MySQL because it's easy to see what's in the DB, both in local and in production. E.g. I looked in the table in my local and found I'd made a mistake with the date formatting and stored just 'C' instead of an actual timestamp before I fixed the code.